MAUI/DataGrid/data-binding.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: Data Binding in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Data Binding support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Data Binding in MAUI DataGrid (SfDataGrid)
+
+The [.NET MAUI DataGrid](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html) Control is bound to an external data source to display the data in the tabular format. It supports data sources such as List, IEnumerable, and so on. The [SfDataGrid.ItemsSource](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_ItemsSourceProperty) property helps to bind this control with collection of objects.
+
+Below codes demonstrate that how to bind a data source to the `SfDataGrid`. Each row in the SfDataGrid would bind to an object in data source. Each column would bind to a property in the data model object.

```

Comment:
The codes below codes demonstrate how to bind a data source to the `SfDataGrid`. Each row in the SfDataGrid binds to an object in the data source. Each column binds to a property in the data model object.

Reply:
Modified.